src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -21,14 +21,18 @@
   @FindBy(css = ".portable-infobox")
   private WebElement infoboxWrapper;
 
-  @FindBy(css = ".portable-infobox-hero-image")
-  private WebElement mainImage;
+  private static final String heroImageCssSelector = ".portable-infobox-hero-image";
+  @FindBy(css = heroImageCssSelector)
+  private WebElement heroImage;
+  private By heroImageSelector = By.cssSelector(heroImageCssSelector);
 
   @FindBy(css = ".portable-infobox .pi-hero-title")
   private WebElement title;
 
-  @FindBy(css = ".portable-infobox .pi-title")
+  private static final String titleSmallImageCssSelector = ".portable-infobox .pi-title";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)